Kconfig

```diff
@@ -2016,6 +2016,7 @@ menu "Library Routines"
 source "libs/libc/Kconfig"
 source "libs/libxx/Kconfig"
 source "libs/libdsp/Kconfig"
+source "libs/rust/Kconfig"
```

Comment:
where rust library source code?

Reply:
The code itself is not yet ready ... I can park this change until this is done.

Reply:
I change it to draft, when it's ready, please mark it ready for review.